charts/pulsar/values.yaml

```diff
@@ -113,6 +115,10 @@ images:
     repository: streamnative/platform
     tag: v1.0.0
     pullPolicy: IfNotPresent
+  presto:
+    repository: eabanonu/presto-pulsar
```

Comment:
any reason why you can't use `apachepulsar/pulsar-all`?

Reply:
I have tried, there is not documentation on how to deploy pulsar sql this way - the reason for this chart.
#33 is the issue I have encountered. 
What has been done in eabanonu/presto-pulsar is to extract pulsar connector into presto plugin folder.

Reply:
Let me think on this, it might be possible to use pulsar-all given the way it is currently been done

Reply:
thank you @eaba!

Reply:
Okay. I am fine with a customized image for now. We will look into how to improve this using apachepulsar/pulsar-all image. Thank you @eaba!

Reply:
Can you remove `.vs` directory? Then I can merge this.